docs/Pflichtenheft.tex

```diff
@@ -213,7 +218,7 @@ \subsubsection{Optionale Funktionalität}
 \item[FA630] Es gibt ein \gls{Dockerimage} zur einfachen Verteilung und isolierten Ausführung der \gls{Uberwachungskonsole}.
 \item[FA640] Bereits registrierte Alarme können wieder gelöscht werden.
 \item[FA650] Das Einstellungsfenster wird um zwei Reiter, "`Verläufe"' und "`Alarme"' erweitert.
-\item[FA660] In den "`Allgemeinen Einstellungen"' ist es dem Nutzer möglich, das Zeitintervall für die Aktualisierung der Werte einzugeben.
+\item[FA660] In den "`Allgemeinen Einstellungen"' ist es dem Nutzer möglich, das Zeitintervall für die Aktualisierung der Werte über einen Schieberegler einzustellen.
```

Comment:
Mit einem Wertebereich von 20 bis 4000 ms ist es schwierig, über einen Schieberegler die exakte Zeit einzustellen. Daher habe ich eine Textbox in die FAs hinzugefügt. UI folgt mit einem anderen Pullrequest.